src/resources/elements/primeDesignSystem/ppopup-modal/ppopup-modal.ts

```diff
@@ -9,12 +9,13 @@ import { AureliaHelperService } from "services/AureliaHelperService";
 export enum ShowButtonsEnum {
   Primary = 0x1,
   Secondary = 0x2,
+  Both = 0x3,
 }
 
 @autoinject
 @customElement("ppopup-modal")
 export class PPopupModal {
-
+  @bindable data: Record<string, unknown>;
```

Comment:
why is `data` needed?  Can we document here?

Reply:
I added a comment